src/data/aws_s3_types.py

```diff
@@ -136,6 +138,29 @@ def to_bucket_data_tagging(tag_response: Dict[str, List[Dict[str, str]]]) -> Buc
     )
 
 
+@dataclass
+class BucketAccessLoggingTagging:
+    ignore_access_logging_check: str = "unset"
+
+
+def to_bucket_access_logging_tagging(tag_response: Dict[str, List[Dict[str, str]]]) -> BucketAccessLoggingTagging:
+    tags = {tag["Key"]: tag["Value"] for tag in tag_response["TagSet"]}
+    ignore_access_logging_check_tag = tags.get("ignore_access_logging_check")
+    ignore_access_logging_check = (
```

Comment:
I think we should lowercase the value first so that values like `TRUE` and `True` still work

Reply:
and perhaps consider returning a bool type to make if statements later a bit cleaner